### PR TITLE
Add Solarized Dark theme

### PR DESCRIPTION
**Pull request by archetypal-one**
Custom theme suggested by @vmavra on OpenChat, based on a terminal color scheme by Ethan Schoonover.

### CODE REVIEW

File: frontend/app/src/theme/community/solarizeddark.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+const txt = lightGray;
+const txtLight = lightishGray;
+
+export function getTheme(base: Theme): Theme {
+    base.name = "solarizeddark";
+    base.label = "Solarized Dark";
+    base.burst = false;
+    base.bg = veryDarkCyan;
+    base.bd = gray;
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/solarizeddark.ts:20]: I think the border is a tiny bit in your face and personally prefer something like 

```
base.bd = "rgba(255,255,255,0.2)";
```

File: frontend/app/src/theme/community/solarizeddark.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    base.txt = txt;
+    base["txt-light"] = txtLight;
+    base.icon.txt = olive;
+    base.icon.selected = olive;
+    base.button.bg = darkBlue;
+    base.button.hv = olive;
+    base.button.txt = txtLight;
```
Comment (julianjelfs) [inline frontend/app/src/theme/community/solarizeddark.ts:27]: To my eyes, the `txtLight` looks a bit clashy with the blue button and actually I reckon `txt` looks a bit nicer.

**Review by julianjelfs — COMMENTED** [review]
Really nice! There are only a couple of things that I think I would prefer. But fairly subjective - what do you think?

**Comment by archetypal-one** [comment]
Agreed! Those small tweaks tie it together nicely. Thanks.

**Review by julianjelfs — APPROVED** [review]

**Pull request merged**